src/QuantumSdk/README.md

```diff
@@ -109,6 +109,9 @@ May contain additional arguments to pass to the Q# command line compiler. Valid
 - `CsharpGeneration`:    
 Specifies whether to generate C# code as part of the compilation process. Setting this property to false may prevent certain interoperability features or integration with other pieces of the Quantum Development Kit. 
 
+- `DefaultSimulator`:
+Specifies the simulator to use by default for execution. Valid values are QuantumSimulator, ToffoliSimulator, ResourcesEstimator, or the fully qualified name of a custom simulator. 
```

Comment:
Currently, the standalone executable treats the resource estimator differently from the other simulators. Do you think it would be better for all of them to be under the `simulate` command, so to display a resource estimate is `simulate --simulator ResourceEstimator`? (I think I'm OK with this idea, the original reason was because the resource estimator has a custom output compared to the other two, but the check for what output to display can be based on the simulator option rather than the simulate/resources commands.)

For custom simulators, I think we might need to clarify that the simulator must be an `IOperationFactory` and has a zero-argument constructor.

Reply:
We can also say here that the default value of `DefaultSimulator` is `QuantumSimulator`.

Reply:
As discussed, let's have the resource estimator listed under the simulator option.